packages/tizen_rpc_port/example/example_proxy/lib/main.dart

```diff
@@ -0,0 +1,141 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
+
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import 'message_proxy.dart';
+
+const String _logTag = 'TizenRpcPortProxyExample';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatefulWidget {
+  const MyApp({super.key});
+
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+String _message = '';
+
+class MyNotify extends NotifyCallback {
+  @override
+  Future<void> onReceived(String sender, String message) async {
+    _message = '$sender: $message';
```

Comment:
Just setting the `_message` value doesn't trigger rebuild of the widgets. `setState` must be called for state changes to take effect. You will need a different architecture for your app.

Also I tried this app on a mobile emulator but the app almost freezes (for about 10 seconds) when I click the send button for the first time. Subsequent calls to `_sendMsg` do not have this problem. Do you have any idea on why?

Reply:
I refactor proxy main.
I created `MessageReceiver` interface.
Can you tell me your opinion?